sites/public/src/components/listings/ListingsList.tsx

```diff
@@ -22,33 +26,48 @@ const ListingsList = (props: ListingsListProps) => {
         <Button>{t("t.clearAllFilters")}</Button>
       </ZeroListingsItem>
     )
+
+  const infoCards =
+    props.currentPage == props.lastPage ? (
+      <div>
+        <InfoCard
+          title={t("t.signUpForAlerts")}
+          subtitle={t("t.subscribeToNewsletter")}
+          className="is-normal-primary-lighter"
+        >
+          <Button className="is-primary">{t("t.signUp")}</Button>
+        </InfoCard>
+        <InfoCard
+          title={t("t.needHelp")}
+          subtitle={t("t.emergencyShelter")}
+          className="is-normal-secondary-lighter"
+        >
+          <Button className="is-secondary">{t("t.helpCenter")}</Button>
+        </InfoCard>
+        <InfoCard
+          title={t("t.housingInSanFrancisco")}
+          subtitle={t("t.seeSanFranciscoListings")}
+          className="is-normal-secondary-lighter"
+        >
+          <LinkButton href="https://housing.sfgov.org/" newTab={true} className="is-secondary">
+            {t("t.seeListings")}
+          </LinkButton>
+        </InfoCard>
+      </div>
+    ) : (
+      <div></div>
+    )
+
   return (
     <div>
       {listingsDiv}
+      <Pagination
+        currentPage={props.currentPage}
+        lastPage={props.lastPage}
+        onPageChange={props.onPageChange}
+      />
       {/* TODO: once pagination is implemented for listings, the following should only show on the last page. */}
```

Comment:
Can you also delete this comment since you addressed the todo?

Reply:
Done